apps/user-profile/src/components/modules/UserOnboardingModal/IntroductionModal/IntroductionModal.js

```diff
@@ -13,11 +13,18 @@ import { onboardingActions } from './../../../../state/actions';
  */
 export default function IntroductionModal() {
     const { onboardingData, dispatch } = useOnboardingDataContext();
-
+    if (!onboardingData || onboardingData.showIntroductionModal === false) {
```

Comment:
Can you move this logic to the `UserOnboardingModal.js` file, its a better place to have this implementation.